### ISSUE TITLE
Build 1.0.0 Broken - @segment/analytics-react-native

### ISSUE THREAD

**Issue by jatsrt** [segmentio/analytics-react-native#75]
Just updated to 1.0.0 and yarn install.  

Seems the package from npm has source but not the built part of the library. 

```
-rw-r--r--   1 jake  staff  14267 Jun  6 11:14 README.md
-rw-r--r--   1 jake  staff    904 Jun  6 11:14 RNAnalytics.podspec
drwxr-xr-x   4 jake  staff    128 Jun  6 11:25 android
drwxr-xr-x   4 jake  staff    128 Jun  6 11:25 ios
-rw-r--r--   1 jake  staff   1957 Jun  6 11:14 package.json
drwxr-xr-x  13 jake  staff    416 Jun  6 11:25 src
```

**Comment by f2prateek** [comment]
hey @jatsrt  - sorry about this! We're fixing this ASAP. In the meantime, I would suggest using the [1.0.1-beta.0](https://github.com/segmentio/analytics-react-native/blob/master/CHANGELOG.md#101-beta0-2019-06-06) version released today.

**Comment by f2prateek** [comment]
This should be fixed in [1.0.1](https://github.com/segmentio/analytics-react-native/blob/master/CHANGELOG.md#101-2019-06-07) - thanks for the report!
